package.json

```diff
@@ -18,7 +18,7 @@
     "build": "yarn run lerna run build",
     "report-coverage": "yarn run lerna run test-coverage",
     "test:watch": "node node_modules/jest/bin/jest.js --watch",
-    "postinstall": "yarn run lerna run postinstall && patch-package && sh scripts/custom_patch_packages.sh && yarn keys:decrypt",
+    "postinstall": "yarn run lerna run postinstall && patch-package && yarn keys:decrypt",
```

Comment:
so we don't need the segment patch anymore?

Reply:
We do, but it's now handled like the other patches. Probably an update to `patch-package` made it handle it properly.

Reply:
Nice!